src/pages/home/sidebar/ChatLinkRow.js

```diff
@@ -85,6 +92,34 @@ const ChatLinkRow = ({
                             </View>
                         )
                     }
+                    {/* For group chat with two or more participants */}
+                    {
+                        Boolean(option.groupIcons && !option.icon && option.groupIcons.length)
```

Comment:
I wonder if we need both `groupIcons` and `icon`? It might be better to combine these into just one `icons` option.

Reply:
Thanks. Fixed. Please check.